doc/changelog.rst

```diff
@@ -18,6 +18,8 @@ Bugfixes:
 - Fix a design flaw in the flag inference mechanism that could very easily
   lead to invalid flags configuration PR #56
 
+- Fix recursion limitations when compiling bytecode with numerous basic
```

Comment:
0.11.0 was already released (I needed the flags fix urgently in another project). Could you move this line to 0.12 ?

Reply:
Okay, will do this.